src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPageFeed/ActionDetailsPageEvent/ActionEventData.tsx

```diff
@@ -14,20 +15,41 @@ const displayName = 'common.ColonyActions.ActionDetailsPage.ActionEventData';
 const ActionEventData = ({
   actionData: { createdAt, transactionHash, type },
   actionData,
+  motionMessageData,
   eventName,
-}: Pick<ActionsPageEventProps, 'actionData' | 'eventName'>) => {
+}: Pick<
+  ActionsPageEventProps,
+  'actionData' | 'eventName' | 'motionMessageData'
+>) => {
   const { colony } = useColonyContext();
+  const { user: motionMessageInitiatorUser } = useUserByNameOrAddress(
+    motionMessageData.initiatorAddress || '',
+  );
+  const messageId = Object.values<string>(ColonyAndExtensionsEvents).includes(
+    eventName,
+  )
+    ? 'event.title'
+    : 'systemMessage.title';
```

Comment:
Could this use `in` operator, as in `eventName in ColonyAndExtensionsEvents ? ... : ...`?